pmaudit

```diff
@@ -490,6 +490,17 @@ def main():
         '-d', '--depsfile',
         metavar='FILE',
         help="save audit data in makefile format to FILE")
+    parser.add_argument(
+        '--shell',
+        metavar='SHELL_COMMAND',
+        help="Name of shell to run (default=SHELL else /bin/sh)")
+    parser.add_argument(
+        '--shellflags',
+        metavar='SHELL_FLAGS',
+        help="Space-separated set of flags to pass to shell (default=-c)")
+    parser.add_argument(
+        '--multiline', action='store_true',
```

Comment:
This logic I need to analyze and think about some more but today is busy so will have to review later.

Reply:
The --multiline is needed to properly support make.  I think it's wise to keep pmaudit/pmash's default "run a command in text" functionality the same as bash/dash, which keep going when a line returns false.  But you want to be able to run *all* the commands in a makefile with a single monitored result.  I think --multiline gets you there.

Reply:
Per other discussions this seems resolved.